src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -37,6 +37,11 @@
 
 public with sharing class BGE_DataImportBatchEntry_CTRL {
 
+    /*******************************************************************************************************
+    * @description The number of additional Data Import Records to load on each infinite scroll query
+    */
+    private static Integer loadSize = 50;
```

Comment:
Yeah, I felt that too. I think we originally said 200 records initially, then loading in batches of 50, but I liked using a single variable here. What about if we load in batches of 200?

Reply:
Have you tried 200? I recall that being slower than I’d like. I’d lean toward having a smaller initial load (50?) and incrementing by a larger number (100?) to avoid jumpiness, but play around with some sample data and see what feels good. :)

Reply:
Sounds like a fair compromise, and I'm sure 200 ends up being pretty slow (though hopefully less so now that we're not counting all the records to get the totals!). I'll give 50/100 load a try!

Reply:
I tried bumped them both up to 100 (initial load and infinite scroll load), and the wait is not visibly more significant than it was with 50 records. Let me know if you test and find otherwise, and I can still trying switching to 50/100 (and we can verify with Steph if that's ok too). 

(...Also it's late in the day and my brain was overcomplicating the 50/100 split, so this is what I've got for now 😅 )